deepfence_frontend/apps/dashboard/src/features/postures/pages/PostureScanResults.tsx

```
@@ -203,13 +208,13 @@ const action = async ({
 
     if (!result.ok) {
       if (result.error.response.status === 403) {
+        const message = await get403Message(result.error);
         return {
           action: actionType,
-          message: 'You do not have enough permissions to delete scan',
+          message,
           success: false,
         };
       }
```

throw error if not 403 everywhere, otherwise errors will be ignored silently.